### ISSUE TITLE
Culturally insensetive name

### ISSUE THREAD

**Issue by dominikwilkowski** [Swastika-IO/Swastika-IO-Core#58]
Hi there.

I am aware that you have another name for the project via #57 but I really do think you should change your name asap if you are in-fact not trying to offend in accordance to your own [code of conduct](https://github.com/Swastika-IO/Swastika-IO-Core/blob/master/CODE_OF_CONDUCT.md).

The Buddhist swastika that you refer to here is actually the one that goes left not right as your logo.
https://en.wikipedia.org/wiki/Swastika

![screen shot 2018-09-04 at 4 53 48 pm](https://user-images.githubusercontent.com/1266923/45014813-242d4e80-b063-11e8-92ff-72f1c3be5cfc.png)
>Left: the left-facing swastika is a sacred symbol in the Bon and Buddhist traditions. Right: the right-facing swastika appears commonly in Hinduism and Jainism.[31][32]

The cultural insensitivity here seems either something that you meant or you're completely ignorant to world history and can't even get your own reference right.

Sorry but this is unbelievably offensive.

I leave some resources here:
https://en.wikipedia.org/wiki/Nazism
https://en.wikipedia.org/wiki/The_Holocaust

Please REALLY read these things. I implore you!

**Comment by Smilefounder** [comment]
Dear @dominikwilkowski ,

I'm really appreciated that you spend your time on our project. Honestly, I'm not meant to ignoring to world history... really. I'm Vietnamese by the way, we also faced a lot of wars. And because of that, I always wish and hope for the peaceful world! More than anyone else. This is the reason why I did choose the name "Swastika", to most Asia peoples, this means 'Well Being', 'Good Existence, and 'Good Luck'. 

And I really hate Hitler that he smeared the beautiful name and its true meaning...

However, we have decided to migrate this project to another new name called "Mixcore" really soon. And we still keep this Swastika I/O project here for the peoples that love and understand the true meaning of this symbol can use.

You will love https://github.com/mixcore the same way we love Swastika I/O for sure. Hope to receive your feedback in the near future.

Best regards.
